### ISSUE TITLE
How to load a large-scale COG dataset hosted on two separate private servers using titiler over MapBox?

### ISSUE THREAD

**Issue by arasharchor** [developmentseed/titiler#149]
I have a large-scale dataset of very big COG images which are hosted on my two private and remote servers (no cloud-based server, no AWS) and the data on each server covers one region. Each image is about 20000x10000px and there are about 5000 of them. 

I intend to load this data from these two separate remote servers on a single MapBox layer using Titiler, but I could not find any info in the docs on this matter.

**Comment by vincentsarago** [comment]
@smajida if you have multiple COGs, you'll need to use virtual `mosaic`. we created [`mosaicJSON`](https://github.com/developmentseed/mosaicjson-spec) to enable this. Titiler is built on top of https://developmentseed.org/cogeo-mosaic/ which enable creating and reading the mosaics. 

We have a notebook showing how to use mosaic: https://developmentseed.org/titiler/examples/Working_with_MosaicJSON/

FYI: I agree there is more to do on the docs 

For your specific architecture, well I don't think I can be much of a help. We tried to make Titiler as simple as possible (see this as a simple python module), the deployment specificities is up to the users. It's true we provide some AWS CDK examples but ... only for example.

Let me know if this help, and feel free to help on the documentation part 🙏
